essentials.yml

```diff
@@ -1,6 +1,29 @@
 ---
+packaging:
+  config:
+    bundle:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      vendor: "Jenkins project"
+      title: "Jenkins Integration Tests WAR"
+      description: "Jenkins WAR for integration tests"
+    war:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      source:
+        dir: "."
 ath:
   useLocalSnapshots: false
   athRevision: "06caf67d37d945ec5228a80d73714fac692c28d3"
   categories:
     - org.jenkinsci.test.acceptance.junit.SmokeTest
+pct:
+  useLocalSnapshots: false
+  jth:
+    version: 2.38
+    passCustomJenkinsWAR: true
+  plugins:
+    - "ssh-slaves"
```

Comment:
You must specify versions here, or the result will be nondeterministic.

Reply:
But what about plugins that are not normally bundled in `jenkins.war`? Testing `mailer`, for example, is not terribly interesting. We should be testing `workflow-job` and so on.

Reply:
We can add specific versions to the WAR using CWP features if needed